content/blog/2023-03-03-expanded-caladapt-enterprise.md

```diff
@@ -0,0 +1,29 @@
+---
+title: An expanded Cal-Adapt enterprise
+author: Cal-Adapt
+image: caladapt_post.png
+tags: other, Fifth Assessment
+snippet: Expanded Cal-Adapt enterprise includes a more comprehensive and powerful solution for highly technical & data-intensive needs, with a focus on the energy sector.
+---
+
+Cal-Adapt is evolving substantially, expanding beyond its current capabilities to support <a href="https://opr.ca.gov/climate/icarp/climate-assessment/" target="_blank">California’s Fifth Climate Change Assessment</a> (Fifth Assessment) and offer a more comprehensive and powerful solution for highly technical & data-intensive needs, with a focus on the energy sector. The expanded Cal-Adapt enterprise will include:
+- A powerful data platform and <a href="https://analytics.cal-adapt.org/" target="_blank">Cal-Adapt Analytics Engine</a>
+- A revamped suite of web tools, analytical tools and computational resources for working with the next generation climate data 
+- A new Historical Observations data platform
+
+The expanded Cal-Adapt enterprise will continue to provide essential visualization, analytical and computational resources to support a broad range of stakeholders in using large climate datasets to inform their decision-making and planning processes. For additional details on state agency vision and roles in this expanded enterprise, please see this <a href="https://efiling.energy.ca.gov/GetDocument.aspx?tn=239123&DocumentContentId=72578" target="_blank">CEC Staff memo</a>.
+
+<img src="/img/blog/caladapt_roadmap.png" style="width:100%;margin:1rem auto;" alt="Evolution of the Cal-Adapt enterprise">
```

Comment:
The image is in apart of the https://github.com/berkeley-gif/caladapt-website-2021/pull/256 update About page PR. So it will be available once everything is merged into the dev branch.